tests/tests_shared.c

```diff
@@ -215,12 +215,12 @@ bool parse_cmd_line(int argc, char** argv, const sir_cl_arg* args, size_t num_ar
     return true;
 }
 
-void wait_for_keypress(void) {
-    (void)printf(WHITEB(EMPH("press any key to exit...")) "\n");
+void wait_for_keypress(void) { // GCOVR_EXCL_START
+    (void)printf(SIR_WHITEB(SIR_EMPH("press any key to exit...%s")), SIR_EOL);
```

Comment:
We need to be consistent with the `SIR_EOL` in the `printf`s. In some you add them after the format string, and in some you add %s and make them an argument. They should all be identical.

Reply:
I did these all automatically with a macro, where it did the easiest thing for the most part.  In this case, it doesn't make sense to have `'COLOR("message\n")'` when we should be using `'COLOR("message")\n'` -- we shouldn't be coloring the newline, so I'll fix that.